### ISSUE TITLE
Build Error from source, module win32verstamp not found

### ISSUE THREAD

**Issue by ONLYA** [mhammond/pywin32#1572]
I have setup the building environment. I build it from source with `python setup.py install`.

It shows that `No module named win32verstamp` during the creation of `pywintypes37`.

I tried it again with `python setup.py install --skip-verstamp`, and it now shows an error in line 1247 in `win32/src/win32job_swig.cpp`, which said:
```
win32/src/win32job_swig.cpp(1247): error C2065: 'JOB_OBJECT_RESERVED_LIMIT_VALID_FLAGS': undeclared identifier
win32/src/win32job_swig.cpp(1247): error C2660: 'PyDict_SetItemString': function does not take 2 arguments
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\cl.exe' failed with exit status 2
```

Both the latest and stable release source codes are tested. Is there possible problem here? Thanks in advance.

**Comment by mhammond** [comment]
I've never seen that problem with win32job, but I suspect you might need a slightly later windows sdk? I've got a WIP patch locally to use a windows 10 SDK - maybe I could push that up as a branch if you think it might help.

**Comment by cleebp** [comment]
We ran into this issue as well for our conda recipe/build upgrade to `pywin32=228`. We found our solution documented deep in `setup.py` where its calling your system level `c:/windows/py.exe` which must have `pywin32` (specifically `win32verstamp`) pip installed into its base environment in order for the build to pass. The try/catch did not bubble up the error message with those details however so tracking the solution down required some digging.

Not sure if this is exactly your problem but hope this helps!
